app/page.tsx

```diff
@@ -95,46 +177,76 @@ export default function Home() {
 					</motion.h1>
 				</ContentSection>
 
-				<ContentSection ref={aboutRef} as={motion.section}>
-					<SectionTitle
-						as={motion.h2}
-						animate={{ opacity: aboutInView ? 1 : 0, y: aboutInView ? 0 : 50 }}
-						transition={{ duration: 0.6, ease: "easeOut" }}
+				<AboutSectionWithSlider
```

Comment:
The about section is cutoff on small phone sizes: 
 
<img width="533" height="777" alt="Image" src="https://github.com/user-attachments/assets/72f76be1-d233-4f7a-a94b-4d16a892afe4" />